README.md

```diff
@@ -1,20 +1,20 @@
 # React Uber Eats
 
 ## Task
-Follow this [Video guide](https://www.youtube.com/playlist?list=PL7FuXFaDeEX0MZnk9tC1tNfjoN4hnc6tl)
+Follow this [Video guide](https://www.youtube.com/playlist?list=PL7FuXFaDeEX0MZnk9tC1tNfjoN4hnc6tl)(p. 1 - 4)
 
 Implement [Uber eats](https://www.ubereats.com/en-UA/feed/?pl=JTdCJTIyYWRkcmVzcyUyMiUzQSUyMkt5aXYlMjIlMkMlMjJyZWZlcmVuY2UlMjIlM0ElMjJDaElKQlVWYTRVN1AxRUFSX2tZQkY5SXhTWFklMjIlMkMlMjJyZWZlcmVuY2VUeXBlJTIyJTNBJTIyZ29vZ2xlX3BsYWNlcyUyMiUyQyUyMmxhdGl0dWRlJTIyJTNBNTAuNDQ5ODU4NyUyQyUyMmxvbmdpdHVkZSUyMiUzQTMwLjUyMzY0OTYlN0Q%3D)
 using [this design](https://www.figma.com/file/cABVXdvRdGiXYzffeOE4gG/Uber-Eats-Adaptive-Dev-Copy?node-id=0%3A1)
 
 Base API url: https://mate-uber-eats-api.herokuapp.com/api/v1/
 
 1. Implement list of restaurants on the `Home` page
-  - use `redux` to store data
-  - use `redux-thunk` to handle async actions
-  - create separate component for restaurant card
-  - use https://mate-uber-eats-api.herokuapp.com/api/v1/restaurants endpoint to fetch data
+    - use `redux` to store data
```

Comment:
I think we should give them tasks doable within a day. So they could have a day goal.

It is also required to split the first task on smaller parts 
So they could implement several small steps rather than one big

Reply:
@mgrinko Agree with you